website/blog/2024-05-22-latest-dbt-stability-improvement-innovation.md

```diff
@@ -16,19 +16,19 @@ As long as dbt Cloud has existed, it has required users to select a version of d
 
 However, this came at a cost. While bumping a project's dbt version *appeared* as simple as selecting from a dropdown, there was real effort required to test the compatibility of the new version against existing projects, package dependencies, and adapters. On the other hand, putting this off meant foregoing access to new features and bug fixes in dbt.
 
-But no more. Today, we're ready to announce the general availability of a new option in dbt Cloud: [**"Versionless."**](https://docs.getdbt.com/docs/dbt-versions/upgrade-dbt-version-in-cloud#versionless)
+But no more. Today, we're ready to announce the general availability of a new option in dbt Cloud: [**"Latest."**](https://docs.getdbt.com/docs/dbt-versions/upgrade-dbt-version-in-cloud#latest)
 
 <!--truncate-->
 
 For customers, this means less maintenance overhead, faster access to bug fixes and features, and more time to focus on what matters most: building trusted data products. This will be our stable foundation for improvement and innovation in dbt Cloud. 
 
-But we wanted to go a step beyond just making this option available to you. In this blog post, we aim to shed a little light on the extensive work we've done to ensure that using "Versionless" is a stable, reliable experience for the thousands of customers who rely daily on dbt Cloud.
+But we wanted to go a step beyond just making this option available to you. In this blog post, we aim to shed a little light on the extensive work we've done to ensure that using "Latest" is a stable, reliable experience for the thousands of customers who rely daily on dbt Cloud.
 
 ## How we safely deploy dbt upgrades to Cloud
 
 We've put in place a rigorous, best-in-class suite of tests and control mechanisms to ensure that all changes to dbt under the hood are fully vetted before they're deployed to customers of dbt Cloud.
 
-This pipeline has in fact been in place since January! It's how we've already been shipping continuous changes to the hundreds of customers who've selected "Versionless" while it's been in Beta and Preview. In that time, this process has enabled us to prevent multiple regressions before they were rolled out to any customers.
+This pipeline has in fact been in place since January! It's how we've already been shipping continuous changes to the hundreds of customers who've selected "Latest" while it's been in Beta and Preview. In that time, this process has enabled us to prevent multiple regressions before they were rolled out to any customers.
```

Comment:
```suggestion
This pipeline has in fact been in place since January! It shows how we've been shipping continuous changes to the hundreds of customers who've selected "Latest" while it's been in Beta and Preview. In that time, this process has enabled us to prevent multiple regressions before they were rolled out to any customers.
```